promscale/page-index/page-index.js

```diff
@@ -0,0 +1,17 @@
+module.exports = [
+  {
+    title: "Promscale",
+    filePath: 'index.md',
+    href: "promscale",
+    name: 'Promscale',
+    excerpt: 'Promscale is an open source long-term store for Prometheus data designed for analytics, built on top of TimescaleDB.',
+    children: [
+      {
+        title: "Promscale",
+        href: "promscale",
+      }
```

Comment:
Much like the discussion in the "update-timescaledb" section, this will result in a URL that is something like: https://docs.timescale.com/promscale/latest/promscale.

Instead, it feels like we should keep up the basic idea of having a "landing" page (index.md) and then relevant child pages (named something differently). In this case I'd:

Create an index.md page which contains the opening paragraph from the `promscale.md` file and then maybe just a separate "Installing Promscale" file with the rest of the content as a child. Thoughts?
```suggestion
    children: [
      {
        href: "install-promscale",
      }
```

Reply:
Sure 👍